api/config/v1alpha1/clusterconfig_types.go

```diff
@@ -114,6 +114,7 @@ type LiqonetConfig struct {
 	//contains a list of reserved subnets in CIDR notation used by the k8s cluster like the podCIDR and ClusterCIDR
 	ReservedSubnets []string               `json:"reservedSubnets"`
```

Comment:
@alacuku Please add the proper description of the members of all the resources, otherwise we may not be able to understand what these are about.